.env.example

```diff
@@ -10,7 +10,7 @@ EMAIL_DOMAIN=
 
 
 # To enable services, add them to the array, like
-ENABLED_SERVICES=oms-global:frontend:core:statutory:events:discounts:mailer
+ENABLED_SERVICES=gateways:frontend:core:statutory:events:discounts:mailer
```

Comment:
if I was a newbie, this would make me confused as from the name `gateway` it's not clear what does it do, maybe we can rename it to something more clear?

Reply:
yeah i was also struggling to find a way to call it, the first thing that came to mind was "ingress", though it would still be unclear (or even less clear, as ingress is a 'keyword' from kubernetes). 

I mean, gateway has use in networking, ingress is a good English word yet possibly less known (and possibly mistaken for k8s stuff?)

What do you propose?

Reply:
is it only aegee portal or something more? if it is, we can name it `portal` or something

Reply:
it is traefik (and kong maybe), so it is a web (and an api) entry point to the application

Reply:
@serge1peshcoff any suggestions on a different name? Otherwise I propose that we merge this and put in the readme what `gateways` contains

Reply:
yeah I've no idea on better naming, let's leave it as it is and rename later if needed.